flask_tutorial-code_review/main.py

```diff
@@ -0,0 +1,71 @@
+from flask import Flask, render_template, request, redirect, url_for, session, flash
+from flask_pymongo import PyMongo
+from datetime import datetime
+from flask_migrate import Migrate
+from config import DevelopmentConfig
```

Comment:
learn how to organzie python imports properly

Reply:
done